conanfile.py

```diff
@@ -13,12 +13,11 @@ class InexorConan(ConanFile):
         "glfw/3.3.2",
         "glm/0.9.9.8",
         "imgui/1.77",
-        "nlohmann_json/3.8.0",
+        "nlohmann_json/3.9.1",
```

Comment:
Should I remove this line entirely? It's pointless as this dependency will be introduced by tinygltf anyways.

Reply:
Yeah I will remove it.